src/NovelRT/Graphics/Vulkan/VulkanGraphicsProvider.cpp

```diff
@@ -242,6 +242,10 @@ namespace NovelRT::Graphics::Vulkan
         createInfo.ppEnabledLayerNames = allValidationLayerPtrs.data();
         createInfo.enabledLayerCount = static_cast<uint32_t>(validationLayerLength);
 
+#ifdef __APPLE__
+        createInfo.flags |= VK_INSTANCE_CREATE_ENUMERATE_PORTABILITY_BIT_KHR;
```

Comment:
Is this for making MoltenVK functional?

Reply:
Yup, same as last explanation about 1.3.216